### ISSUE TITLE
Question on the doc of canonical-addrss

### ISSUE THREAD

**Issue by lucidfrontier45** [rethinkdb/rethinkdb#5853]
The current document says

```
--canonical-address addr: address that other RethinkDB instances will use to connect to us, can be specified multiple times
```

Can I specify both address and port like `--canonical-address 1.2.3.4:39015`?
If so, `addr` in the above doc should be modified to `host:port` just like the `-j` option.

```
-j [ --join ] host:port: host and port of a RethinkDB node to connect to
```

**Comment by danielmewes** [comment]
Interesting catch, thanks for bringing it up.

I thought myself that it wasn't possible to specify a port for `canonical-address`, but after taking a brief look at the server code I think that you're right and it's indeed possible to provide a port as well.

We'll update the help output and also the documentation for the config file.

**Comment by danielmewes** [comment]
The documentation is fixed, but the `--help` output still needs to be updated.

**Comment by srh** [comment]
Fixed the help message with #6576.
